Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py

```diff
@@ -190,9 +190,14 @@ def get_report_triage(client: Client, **args) -> CommandResults:
     """
     sample_id = args.get("sample_id")
     task_id = args.get("task_id")
+    if not task_id.startswith("behavioral"):
```

Comment:
```suggestion
    if not str(task_id).startswith("behavioral"):
```
Regarding the build failures, please cast it to a string.